en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies.md

```diff
@@ -0,0 +1,114 @@
+# Managing Common API Policies
```

Comment:
Let's add a small description under this to introduce "common API policies" and a summary of the ways we can manage them.